app/src/pages/inside/dashboardItemPage/newWidgets/charts/launchesComparisonChart/launchesComparisonChart.scss

```diff
@@ -0,0 +1,58 @@
+.legend {
+  position: absolute;
+  top: 5px;
+  left: 0;
+  right: 0;
+  text-align: right;
+  padding: 20px 20px 0;
+}
+
+.color-mark {
+  display: inline-block;
+  width: 12px;
+  height: 12px;
+  margin-right: 1px;
+  border-radius: 6px;
+  position: relative;
+  top: 2px;
+
+  &:before {
+    position: absolute;
+    display: none;
+    left: 2px;
+    top: 2px;
+    width: 8px;
+    height: 8px;
+    border-radius: 100%;
+    background-color: #fff;
```

Comment:
It's not allowed to define colors by hex codes. Please use existing color variables.

Reply:
Check for other cases.

Reply:
Fixed